lib-find-meal/src/main/java/com/kinandcarta/lib/find/meal/adapter/MealsAdapter.kt

```diff
@@ -55,8 +54,9 @@ class MealsAdapter(var mealsList: List<Meal>) :
             // TODO calculate distance (this could be done in the shared layer)
             distanceView.text = "8637.35km"
 
-            availableView.text = "Available: ${parseDate(meal.availableFromDate)}"
-            expiryView.text = "Expires: ${parseDate(meal.expiryDate)}"
+            // TODO format these dates
+            availableView.text = "Available: ${meal.availableFromDate}"
+            expiryView.text = "Expires: ${meal.expiryDate}"
```

Comment:
Changing the dates to unix timestamp broke the parsing 😬 Could be worth changing the date formatting to happen in the shared code so that we can just display the `availableFromDate` and `expiryDate` without messing about with the Meal object we get back? Something to think on or discuss at some point